packages/babel-traverse/src/path/lib/virtual-types.js

```diff
@@ -113,6 +113,8 @@ export let Flow = {
       return node.importKind === "type" || node.importKind === "typeof";
     } else if (t.isExportDeclaration(node)) {
       return node.exportKind === "type";
+    } else if (t.isImportSpecifier(node)) {
```

Comment:
Oops, I assumed it was failing due to the parser -- didn't look closely. Thanks for the heads up!